TeamCode/src/main/java/org/firstinspires/ftc/teamcode/ftc16072/QQTest/TestMotor.java

```diff
@@ -0,0 +1,37 @@
+package org.firstinspires.ftc.teamcode.ftc16072.QQTest;
+
+import com.qualcomm.robotcore.hardware.DcMotor;
+
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+
+import java.util.jar.Attributes;
+
+public class TestMotor extends QQtest {
+    DcMotor motor;
+    double speed;
+
+    public TestMotor(String name, double speed, DcMotor Motor) {
+        super(name);
+        this.speed = speed;
+        this.motor = motor;
+
+    }
+
+
+
+
+
+    public void run(boolean on, Telemetry telemetry){
+        if (on=true){
```

Comment:
a single = is an assignment, so this will set on to be true.   What you really want is the simpler:
`if(on){`

Reply:
fixed